lua/examples/example_of_dynamic_config_request.lua

```diff
@@ -40,21 +40,14 @@ dylib.add_grantor_entry_lb("198.51.100.0/25", addrs, dyc.gk)
 addrs[1] = { gt_ip = '203.0.113.4', gw_ip = '10.0.2.250' }
 dylib.update_grantor_entry_lb("198.51.100.0/25", addrs, dyc.gk)
 
--- Examples of temporarily changing global and block log levels.
-local old_log_level = staticlib.c.rte_log_get_global_level()
-staticlib.c.rte_log_set_global_level(staticlib.c.RTE_LOG_ERR)
+-- Examples of temporarily changing block log levels.
 
 local cpsc = staticlib.c.get_cps_conf()
 if cpsc == nil then
 	return "cps: failed to fetch config to update log level"
 end
```

Comment:
Given that `cpsc.log_type` is going away, can't we drop these lines of code as well?